<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileProperty</class>
 <widget class="QDialog" name="FileProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="fileName">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>FileName</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="timestamp">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sizeLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件大小：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="fileType">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="icon">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>上传时间：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="fileSize">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件类型：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
